spec/models/comment_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  # tests go here
+

```

Comment:
- [ ] Kindly add some of the validation test case here ( +ve & -ve ) cases also

Reply:
Please observe this file closely. I believe I do have a few test cases written in the file 😅 
But in any case, I will surely take a good look at the file again. Thank you very much for the great review.